README.rst

```diff
@@ -20,15 +20,35 @@ Super CSV is a stand alone library that can be used for CSV management, both syn
 
 Testing
 -------
-
 ::
+
     $make requirements
-Will install all pre-requisuites.
 
-::
+Will install all pre-requisuites. ::
+
     $make test
+
 Runs the unit tests in local environment.
 
+Using with Docker Devstack
+--------------------------
+Prerequisite: Have your Open edX https://github.com/edx/devstack properly installed.
+Note: When you see "from inside the lms" below, it means that you've run ``make lms-shell`` from your devstack directory and are on a command prompt inside the LMS container.
+
+#. Clone this repo into ../src/ directory (relative to your "devstack" repo location). This will mount the directory in a way that is accessible to the lms container. Since this is a standalone library, there should be no need to manually perform a pip-install step in the lms-shell.
```

Comment:
> Since this is a standalone library, there should be no need to manually perform a pip-install step in the lms-shell.

I'm not confident that's correct. Do you have a line in your `requirements/private.txt` which is making it unnecessary to manually `pip install -e` this package?

You may find want to try cloning a new `devstack` and trying to get set up with only these instructions, and see where you get stuck.

Reply:
The reason for the above lines is that this library is referenced by bulk-grades, so when bulk grades are installed, this will also get pulled in. What would be better phrasing? Just removing the caveat.

Reply:
The version of super-csv that will be pulled in by default if you just leave it to what edx-bulk-grades requires is pulled in from PyPI; if you wanted to test potential changes to super-csv in a devstack LMS, you'd need to install the local version of super-csv one way or another.

You do get super-csv for free by virtue of edx-bulk-grades being installed, but that doesn't help folks get set up for testing whether code changes do what they expect. Does that make sense?